scripts/download-prebuilt.sh

```diff
@@ -0,0 +1,84 @@
+#!/bin/bash
+
+set -e
+
+SETUP_SRC=$(realpath ${BASH_SOURCE[@]})
+SETUP_DIR=$(dirname $SETUP_SRC)
+TOP_DIR=$(realpath $SETUP_DIR/..)
+BUILD_DIR=$TOP_DIR/build
+PREBUILT_DIR=$BUILD_DIR/prebuilt
+
+if [ ! -e $BUILD_DIR ]; then
+	echo "$BUILD_DIR does not exist so it looks like the prerequisit tools may not be available"
+	echo "They can be installed by running scripts/get-env-root.sh and scripts/get-env.sh"
+	exit 1
+fi
+
+: ${BOARD:="atlys"}
+: ${TARGET:="hdmi2usb"}
+: ${PREBUILT_RELEASE:="unstable"}
+: ${PREBUILT_REPO:="timvideos/HDMI2USB-firmware-prebuilt"}
+
+
+if [ "$TARGET" = "hdmi2usb" ]; then
+	GATEWARE="${BOARD}_hdmi2usb-hdmi2usbsoc-${BOARD}.bit"
+elif  [ "$TARGET" = "base" ]; then
+	GATEWARE="${BOARD}_base-minisoc-${BOARD}.bit"
+fi
+
+FIRMWARE="hdmi2usb.hex"
+
+GATEWARE_DIR="$TOP_DIR/third_party/misoc/build"
+FIRMWARE_DIR="$TOP_DIR/firmware/fx2"
+
+# Check nothing exists before we start
+if [ -e $GATEWARE_DIR/$GATEWARE ]; then
+	echo ""
+	echo "It looks like a build has been run before"
+	echo "Run make clean then try again"
+	echo ""
+	exit 1
+elif [ -e $FIRMWARE_DIR/$FIRMWARE ]; then
+	echo ""
+	echo "It looks like a build has been run before"
+	echo "Run make clean then try again"
+	echo ""
+	exit 1
+elif [ -e $PREBUILT_DIR ]; then
+	echo ""
+	echo "It looks like a download has been run before"
+	echo "Run make clean then try again"
+	echo ""
+	exit 1
+fi
+
+LATEST_BUILD=$(wget https://raw.githubusercontent.com/${PREBUILT_REPO}/master/${BOARD}/firmware/${PREBUILT_RELEASE} -O- -q)
+
+BASE_URL="https://github.com/${PREBUILT_REPO}/raw/master/${BOARD}/firmware/${LATEST_BUILD}"
+
+SHA256SUM_FILE="sha256sum.txt"
+
+(
+	mkdir $PREBUILT_DIR
+	cd $PREBUILT_DIR
+	echo ""
+	echo "BOARD=$BOARD TARGET=$TARGET PREBUILT_RELEASE=$PREBUILT_RELEASE"
+	echo "Downloading files from https://github.com/${PREBUILT_REPO} ..."
+	echo ""
+	wget -nv ${BASE_URL}/${GATEWARE}
+	wget -nv ${BASE_URL}/${FIRMWARE}
+	wget -nv ${BASE_URL}/${SHA256SUM_FILE}
```

Comment:
Could you download the sha256sum file and then just download every file listed in it?